src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -30,8 +30,13 @@ foam.CLASS({
     'foam.u2.TableView'
   ],
 
+  imports: [
+    'window'
+  ],
+
   constants: {
-    CSS_NAME: 'foam-u2-search-FilterController'
+    CSS_NAME: 'foam-u2-search-FilterController',
```

Comment:
Combining all the CSS from the three models (`PolymerController`, `Controller` and `FilterController` under the same name. Otherwise it would only pick up the CSS for the childmost class, and not have the parents'.

I agree it sucks, but we don't have a better plan right now.